akka-remote/src/main/scala/akka/remote/transport/netty/NettySSLSupport.scala

```diff
@@ -80,24 +76,18 @@ private[akka] class SSLSettings(config: Config) {
       case e: GeneralSecurityException ⇒ throw new RemoteTransportException("Server SSL connection could not be established because SSL context could not be constructed", e)
     }
 
-  def createSecureRandom(log: MarkerLoggingAdapter): SecureRandom = {
+  private[remote] def createSecureRandom(): SecureRandom = {
     val rng = SSLRandomNumberGenerator match {
       case r @ ("AES128CounterSecureRNG" | "AES256CounterSecureRNG") ⇒
-        log.debug("SSL random number generator set to: {}", r)
         SecureRandom.getInstance(r, AkkaProvider)
       case s @ ("SHA1PRNG" | "NativePRNG") ⇒
-        log.debug("SSL random number generator set to: {}", s)
         // SHA1PRNG needs /dev/urandom to be the source on Linux to prevent problems with /dev/random blocking
         // However, this also makes the seed source insecure as the seed is reused to avoid blocking (not a problem on FreeBSD).
         SecureRandom.getInstance(s)
 
-      case "" ⇒
-        log.debug("SSLRandomNumberGenerator not specified, falling back to SecureRandom")
-        new SecureRandom
+      case ""      ⇒ new SecureRandom
 
-      case unknown ⇒
-        log.warning(LogMarker.Security, "Unknown SSLRandomNumberGenerator [{}] falling back to SecureRandom", unknown)
-        new SecureRandom
+      case unknown ⇒ throw new IllegalArgumentException(s"Unknown PRNG configured '$unknown'. Please change the setting at akka.remote.netty.ssl.random-number-generator.")
```

Comment:
Couldn't this change make it a non-drop-in artifact, worked before but now it fails?